src/extension/background-script/connectors/lnd.ts

```diff
@@ -81,7 +83,58 @@ class Lnd implements Connector {
       };
     });
   }
+  async sendPaymentKeySend(
+    args: SendPaymentArgs
+  ): Promise<SendPaymentResponse> {
+    //See: https://gist.github.com/dellagustin/c3793308b75b6b0faf134e64db7dc915
+    const byteArray = new Uint8Array(32);
 
+    const dest_pubkey_hex = args.pubkey;
+    const dest_pubkey_base64 = Buffer.from(dest_pubkey_hex, "hex").toString(
+      "base64"
+    );
+    console.log(dest_pubkey_base64);
+
+    for (let i = 0; i < 32; i++) {
+      const randomNumber = _.random(0, 255);
+      byteArray[i] = randomNumber;
+    }
+
+    const encoded = Buffer.from(byteArray).toString("base64");
+    const hash = CryptoJS.enc.Base64.stringify(
+      CryptoJS.SHA256(CryptoJS.enc.Base64.parse(encoded))
+    );
+    return this.request<{
+      payment_preimage: string;
+      payment_hash: string;
+      payment_route: { total_amt: number; total_fees: number };
+      payment_error?: string;
+    }>(
+      "POST",
+      "/v1/channels/transactions",
+      {
+        dest: dest_pubkey_base64,
+        amt: args.amount,
+        payment_hash: hash,
+        dest_custom_records: {
```

Comment:
can we allow passing in of this custom records? that developers can decide what records to set? 

e.g. I think the podcasting uses a special one: https://github.com/lightning/blips/blob/master/blip-0010.md